xarray/tests/test_dataarray.py

```diff
@@ -1784,6 +1784,24 @@ def test_reduce(self):
                              dims=['x', 'y']).mean('x')
         assert_equal(actual, expected)
 
+    def test_reduce_dtype(self):
+        coords = {'x': [-1, -2], 'y': ['ab', 'cd', 'ef'],
+                  'lat': (['x', 'y'], [[1, 2, 3], [-1, -2, -3]]),
+                  'c': -999}
+        orig = DataArray([[-1, 0, 1], [-3, 0, 3]], coords, dims=['x', 'y'])
+
+        for dtype in [np.float32, np.float64, np.float128]:
```

Comment:
Looks like you'll need to remove `np.float128`. It appears that it's not available on Windows, which is triggering this test failure.